vault/resource_okta_auth_backend.go

```diff
@@ -263,12 +283,15 @@ func oktaAuthBackendRead(d *schema.ResourceData, meta interface{}) error {
 		return nil
 	}
 
+	d.Set("path", path)
```

Comment:
Should we be checking the error return values for these `Set` calls?

Reply:
Good question, taking a look now.

Reply:
Fixed in [8c19db6](https://github.com/hashicorp/terraform-provider-vault/pull/1123/commits/8c19db6d726cbbf8156ba2105bd84ef0dd293f6a)